builder/vmware/common/driver_esx5.go

```diff
@@ -28,11 +28,11 @@ import (
 	"github.com/vmware/govmomi/vim25/types"
 
 	"github.com/hashicorp/go-getter/v2"
-	"github.com/hashicorp/packer/communicator/ssh"
 	"github.com/hashicorp/packer/helper/communicator"
 	"github.com/hashicorp/packer/helper/multistep"
 	helperssh "github.com/hashicorp/packer/helper/ssh"
 	"github.com/hashicorp/packer/packer"
+	"github.com/hashicorp/packer/sdk-internals/communicator/ssh"
```

Comment:
I think you probably already know that but I'm stating it in case: if that package was under internal  like in `github.com/hashicorp/packer/internal/sdk`  then only the packages from `github.com/hashicorp/packer`  (and tree) could import it. More here: https://golang.org/doc/go1.4#internalpackages